app/controllers/application_controller.rb

```diff
@@ -8,9 +8,16 @@ class ApplicationController < ActionController::Base
   before_action :set_user
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :check_authorization
+  before_action :check_max_session_time
 
 protected
 
+  def check_max_session_time
+    return false if current_user.nil?
+    time_out = Rails.application.secrets.session_expiry_in_minutes.nil? ? 60 : Rails.application.secrets.session_expiry_in_minutes.to_i
```

Comment:
it doesn't need to be in secrets, can we define that as a config value and put the default there, instead of here?